test/tests/userobjects/layered_integral/tests

```diff
@@ -28,7 +28,15 @@
       input = 'average_sample.i'
       exodiff = 'average_sample_out.e'
 
-      detail = 'as an average calculation.'
+      detail = 'as an average calculation,'
```

Comment:
I do not understand what the detail writer on these tests was trying to do.  These detail fragments don't mesh well with the group prefix. @aeslaughter - are detail comments suppose to not "finish" the group sentence each?  Why do these end with "with" and "and"?

Reply:
The requirement just needs to be cleaned up. I tend to write them as one complete sentence if possible.

```
requirement = "The system shall output data the"
detail = "ExodusII,"
detail = "CSV, and"
detail = "JSON formats."
```

This will render as:

The system shall output data the
(a) ExodusII,
(b) CSV, and
(c) JSON formats.

Reply:
the way I wrote the requirements should work either way?